internal/collector/filter.go

```diff
@@ -23,9 +23,14 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+const (
+	eventReasonMask = "ReasonMaskedByExporter"
```

Comment:
We already know this is reason as it will be in the reason label so think that is not needed, still think just Masked is good enough, but up to you.